src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile

```diff
@@ -98,9 +99,8 @@ ENV RUST_CONFIGURE_ARGS \
       --set llvm.thin-lto=true \
       --set llvm.ninja=false \
       --set rust.jemalloc
-ENV SCRIPT python2.7 ../x.py dist --host $HOSTS --target $HOSTS \
-        --include-default-paths \
-        src/tools/build-manifest
+COPY host-x86_64/dist-x86_64-linux/pgo.sh /tmp/
+ENV SCRIPT /tmp/pgo.sh
```

Comment:
While in the long term we'll probably want some sort of `build.pgo = true` in `config.toml`, I think there is a quick improvement we can make to have the PGO setup be more reusable:

```dockerfile
ENV PGO_HOST x86_64-unknown-linux-gnu

ENV SCRIPT /tmp/pgo.sh python2.7 ../x.py dist --host $HOSTS --target $HOSTS \
        --include-default-paths \
        src/tools/build-manifest
```

`pgo.sh` would then compile the compiler for `PGO_HOST`, run the sample compilations with it and invoke:

```sh
$@ --rust-profile-use=/tmp/rustc-pgo.profdata
```

With a setup like this we could move `pgo.sh` outside of the docker directory and almost effortlessly apply it to other platforms (when we get CI time budget for that).